data/gegno/engines.txt

```diff
@@ -0,0 +1,225 @@
+outfit "RG15 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 26000
+	mass 31
+	thumbnail "outfit/small torch"
+	"engine capacity" -31
+	"outfit space" -31
+	thrust 7.67
+	"thrusting energy" 0.9
+	"thrusting heat" 2
+	"flare sprite" "effect/ion torch/small"
+		"frame rate" 2.5
+	"flare sound" "ion torch small"
+	description "The RG15 Ion Torch Thruster is the smallest torch thruster found amongst Gegno Vi ships and it is almost exclusively mounted on the Slate. It's not amazingly efficient, weighing more and producing more heat than most thrusters of a similar size, but also has a slightly lower energy consumption rate."
+
+outfit "RG15 Torch Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 21000
+	mass 25
+	thumbnail "outfit/small torch steer"
+	"engine capacity" -25
+	"outfit space" -25
+	turn 205
+	"turning energy" 0.4
+	"turning heat" 1.4
+	"steering flare sprite" "effect/ion torch/small"
+		"frame rate" 2.4
+	"steering flare sound" "ion torch small"
+	description "This steering system that incorporates RG15 thrusters is only meant for the smallest of Gegno Vi ships, and mounting it on anything larger than a Shuttle-class ship is unheard of. Despite this, its energy requirement is a notch lower than normal, so some Gegno pilots use it as a cheap addon."
+
+outfit "RG18 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 58000
+	mass 51
+	thumbnail "outfit/med torch"
+	"engine capacity" -51
+	"outfit space" -51
+	thrust 14.8
+	"thrusting energy" 1.2
+	"thrusting heat" 3.5
+	"flare sprite" "effect/ion torch/medium"
+		"frame rate" 2.4
+	"flare sound" "ion torch medium"
+	description "Ion Torch Thrusters are the Gegno's equivalent of humanity's ion thrusters, but are still earlier in design and lacking in most departments when compared. They have heavier frames and their thrust output is a lot more chaotic and explosive, causing them to run notably hotter but require slightly less energy. The RG18 is the most common-sized thruster amongst Gegno Vi vessels, and a notable size at that."
+
+outfit "RG18 Torch Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 46000
+	mass 39
+	thumbnail "outfit/med torch steer"
+	"engine capacity" -39
+	"outfit space" -39
+	turn 420
+	"turning energy" 1.3
+	"turning heat" 2.4
```

Comment:
I think these should consume less energy, around 0.9, and produce more heat, around 2.7.
![image](https://user-images.githubusercontent.com/20605679/173210505-43656c17-b755-47a3-ae6d-27a508f5f43e.png)
The turn/ton may also be a bit high. If the turn is reduced, the energy and heat should also be reduced with that. If the size is increased, then I'd also argue for higher efficiency.

Reply:
This falls in-line with what I said about the discrepancies between the RG18 and RG3, but you're right in that the turn does seem high in comparison, and especially to that philosophy. Changed the values to the ones you gave me, and am going to take the turning down to 408. Does that suffice?

Reply:
Sounds good. I think 0.75 will do as to not go too far.